### DISCUSSION TITLE
[Feedback] New GMod chat box

### DISCUSSION THREAD

**Discussion by Derpius** [TAServers/issues#10] [Announcements]
## ❓ What

We've just added a custom chat box addon to the server, which you can find here: https://github.com/StyledStrike/gmod-custom-chat.

~~We're going to fork it to add features more tailored to TAS and our existing chat relay.~~ We've forked it to https://github.com/TAServers/gmod-custom-chat

## ❓ Why

The default GMod chat box is awful. With this we get a number of features:

- Image embeds
- Emotes
- Profile icons next to messages
- More message formatting options
- User customisation
- And more

## 🗣️ What we want to hear from you

- Do you have any complaints about the chat box at the moment?
- Do you think it's a worthwhile improvement?
- Do you have any suggestions of what we should change/add?
  - We're already planning on integrating our Discord emotes for example

**Comment by ELPatitoLoco** [comment]
Its a good addon, maybe it would be cool to have a pm tab, and a settings tab like better chat does, also would be cool to have a option to blur images from discord or whatever site because you can send pretty much anything including gore and nsfw.

**Reply by Derpius** [reply]
It should have some degree of customisation in the settings menu (accessible from the context menu)

**Comment by Wr1Puppy** [comment]
+1 anything is better than gmods shit

**Comment by Kilometres** [comment]
Large improvement from the default chat box. More customisation would be nice, the current font and font size cause the bottom of the text to get cut off for me. (2560x1400 at 100% Scaling)

**Reply by Derpius** [reply]
Yeah I have cutoff issues on my display as well (1440p)

**Reply by ELPatitoLoco** [reply]
mf's got 4k
:)
